website/content/docs/agent-and-proxy/agent/generate-config/index.mdx

```diff
@@ -0,0 +1,110 @@
+---
+layout: docs
+page_title: agent generate-config - Command
+description: |-
+  Generates a simple Vault Agent configuration file from the given parameters.
+---
+
+# agent generate-config
+
+Generates a simple Vault Agent configuration file from the given parameters.
+
+Currently, the only supported configuration type is `env-template`, which
+helps you generate a configuration file with environment variable templates
+for running Vault Agent in process supervisor mode.
```

Comment:
I know that this is a bit of a chicken and egg situation, but it'd be great if this (and other parts of the page) linked to the future page for the process supervisor subsystem. We could establish those links once we add the docs about the process supervisor mode?